sonic-chassisd/scripts/chassisd

```diff
@@ -244,7 +245,10 @@ class ModuleUpdater(logger.Logger):
             module_info_dict = self._get_module_info(module_index)
             if module_info_dict is not None:
                 key = module_info_dict[CHASSIS_MODULE_INFO_NAME_FIELD]
-
+                # initialize for the first time
+                if key not in self.module_status:
+                    self.module_status[key] = "Init"
```

Comment:
Lets fetch the prev status from the Db instead of storing it locally.

Reply:
> Lets fetch the prev status from the Db instead of storing it locally.

Per suggestion, define get_module_curretn_status() to fetch the previous status from DB instead of creating moduel_status[]

Reply:
@deepak-singhal0408  Please review the new change